src/makeRobot.js

```diff
@@ -38,7 +38,59 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
коли імя змінної таке є як ключ можна використати скорочений запис
```suggestion
    name,
    wheels,
    version,
```